srcpkgs/python3-pyside2/template

```diff
@@ -18,34 +18,59 @@ makedepends="python3-devel libshiboken2-devel qt5-devel
  qt5-serialport-devel
  $(vopt_if webengine 'qt5-webengine-devel')"
 depends="python3-shiboken2"
+checkdepends="python3-distro tzdata pulseaudio qt5-quickcontrols"
 short_desc="Python3 bindings for the Qt5 toolkit"
 maintainer="yopito <pierre.bourgin@free.fr>"
 license="LGPL-3.0-or-later"
 homepage="https://wiki.qt.io/Qt_for_Python"
 distfiles="https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-${version}-src/${_pkgname}.tar.xz"
-checksum=f1cdee53de3b76e22c1117a014a91ed95ac16e4760776f4f12dc38cd5a7b6b68
+checksum=b306504b0b8037079a8eab772ee774b9e877a2d84bab2dbefbe4fa6f83941418
+
+build_options="webengine"
+desc_option_webengine="Build Qt5 WebEngine bindings"
+
+subpackages="libpyside2-python3-devel libpyside2-python3"
 
 # needed by shiboken2 runtime
-export CLANG_INSTALL_DIR=${XBPS_CROSS_BASE}/usr
+export CLANG_INSTALL_DIR=/usr
 
 if [ ${CROSS_BUILD} ]; then
 	hostmakedepends+=" qt5-qmake qt5-host-tools qt5-tools qt5-remoteobjects qt5-scxml"
 fi
 
-build_options="webengine"
-desc_option_webengine="Build Qt5 WebEngine bindings"
-
 if [ "$XBPS_TARGET_ENDIAN" = "le" ]; then
-	build_options_default="webengine"
+	if [ "$XBPS_TARGET_WORDSIZE" = "$XBPS_WORDSIZE" ]; then
+		build_options_default="webengine"
+	fi
 fi
 
-subpackages="libpyside2-python3-devel libpyside2-python3 libpyside-python3
- python3-pyside python3-pyside-phonon python3-pyenet"
-
 if [ "$build_option_webengine" ]; then
 	subpackages+=" python3-pyside2-webengine"
 fi
 
+pre_build() {
+	if [ ${CROSS_BUILD} ]; then
+
+		# make clang of shiboken aware of crossbuild
+		SHIBOKEN_CLANG_OPTIONS="--target=${XBPS_CROSS_TRIPLET}"
+		SHIBOKEN_CLANG_OPTIONS+=" --sysroot=/usr/${XBPS_CROSS_TRIPLET}"
+
+		# qt5 is built with gcc, so shiboken's clang needs gcc corresponding headers
+		local gcc_ver="$(${XBPS_CROSS_TRIPLET}-gcc -dumpversion)"
+		local inc_root="/usr/${XBPS_CROSS_TRIPLET}/usr/include/c++/${gcc_ver}"
```

Comment:
Could you add a comment about `backward` so if it breaks we remember to fix?

Reply:
yes, sure. here it is.

Reply:
just include backwards always even if it's not needed, it's a standard include so it should be there, it won't hurt anything if it's there